tests/tests_unit/test_api/test_labels.py

```diff
@@ -29,28 +29,43 @@ def test_list(self, mock_labels_response):
         assert "P" == jsgz_load(mock_labels_response.calls[0].request.body)["filter"]["externalIdPrefix"]
         assert mock_labels_response.calls[0].response.json()["items"] == res.dump(camel_case=True)
 
+    def test_access_properties(self, mock_labels_response):
+        res = LABELS_API.list(external_id_prefix="P")
+        assert res[0].name == "Pump"
+        assert res[0].description == "guess"
+        assert res[0].external_id == "PUMP"
+        assert res[0].created_time > 0
+
     def test_call(self, mock_labels_response):
         list(LABELS_API(limit=10))
         calls = mock_labels_response.calls
         assert 1 == len(calls)
         assert {"cursor": None, "limit": 10, "filter": {}} == jsgz_load(calls[0].request.body)
 
     def test_create_single(self, mock_labels_response):
-        res = LABELS_API.create(Label(external_id="1"))
-        assert isinstance(res, Label)
+        res = LABELS_API.create(LabelDefinition(external_id="1", name="my_label", description="mandatory"))
```

Comment:
I think this is going to give persistent support questions with people using Label.
Should we use Label for this and sth like AttachedLabel for the other?

Reply:
> I think this is going to give persistent support questions with people using Label.

What do you mean here?

Reply:
people will .create(Label()) and then ask support

Reply:
Hm, this is at least consistent with the API. I can rename labels API to label definition api

Reply:
I added a type check for LabelDefinition and raise an exception if they use(d) Label